code/modules/shareddream/dream_entry.dm

```diff
@@ -29,18 +48,33 @@ var/list/dream_entries = list()
 		if(istype(bg) || force_wakeup)
 			// If we choose to be asleep, keep sleeping.
 			if(willfully_sleeping && sleeping && stat == UNCONSCIOUS)
-				sleeping = 5
-				return
-			for(var/thing in SSpsi.processing)
-				var/datum/psi_complexus/psi = thing
-				to_chat(psi.owner, SPAN_CULT("You feel the activity of Srom decrease."))
```

Comment:
I think these lines were deleted by mistake - should probably be moved to 54 where the admin msg is